webapp/templates/base.html.twig

```diff
@@ -1,22 +1,52 @@
 <!DOCTYPE html>
 <html>
-    <head>
-        <meta charset="UTF-8">
-        <title>{% block title %}Welcome!{% endblock %}</title>
-        <link rel="icon" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 128 128%22><text y=%221.2em%22 font-size=%2296%22>⚫️</text></svg>">
-        
-        <link rel="preload" as="style" href="/styles/app.css" />
-        <link rel="stylesheet" href="/styles/app.css" />
-
-        {% block stylesheets %}
-            {{ encore_entry_link_tags('app') }}
-        {% endblock %}
-
-        {% block javascripts %}
-            {{ encore_entry_script_tags('app') }}
-        {% endblock %}
-    </head>
-    <body>
-        {% block body %}{% endblock %}
-    </body>
+	<head>
+		<meta charset="UTF-8">
+		{% block meta %}{% endblock %}
+		<title>
+			{% block title %}{% endblock %}
```

Comment:
Maybe you could choose 'Labyrinth' as page title, or something fancier like marmelabyrinth 😁